test/functional/p2p_addr_relay.py

```diff
@@ -282,7 +276,8 @@ def getaddr_tests(self):
         block_relay_peer.send_and_ping(msg_getaddr())
         inbound_peer.send_and_ping(msg_getaddr())
 
-        self.mocktime += 5 * 60
+        # invoke m_next_addr_send timer, see above for rationale
```

Comment:
This line does not clearly explain where to look for the rationale. Maybe mentioning the function name (where it has been described) might help.

```suggestion
        # invoke m_next_addr_send timer, see the rationale under definition of send_addr_msg() function
```

Reply:
done